src/components/Session/TrustButton.jsx

```diff
@@ -50,7 +50,11 @@ export default class TrustButton extends React.Component {
       }
     } else {
       if (found) {
-        button = <span className="AddTrustRow__exists">{this.props.message}</span>
+        if (this.props.message.startsWith("http://") || this.props.message.startsWith("https://")) {
```

Comment:
Lets not link to sites with `http://`. Sites that use `http://` don't deserve to get linked to

Reply:
ok, will update this as well.